### PR TITLE
[sonos] Add discovery of IKEA SYMFONISK

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.sonos/src/main/resources/ESH-INF/thing/SYMFONISK.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+			<channel id="zonegroupid" typeId="zonegroupid" />
+			<channel id="zonename" typeId="zonename" />
+			<channel id="coordinator" typeId="coordinator" />
+			<channel id="sleeptimer" typeId="sleeptimer" />
+			<channel id="currenttransporturi" typeId="currenttransporturi" />
+			<channel id="currenttrackuri" typeId="currenttrackuri" />
+		</channels>
+
+		<properties>
+			<property name="vendor">SONOS</property>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.sonos/src/main/resources/ESH-INF/thing/SYMFONISK.xml]: Is the vendor Sonos or IKEA...?

Comment (lolodomo) [reply bundles/org.openhab.binding.sonos/src/main/resources/ESH-INF/thing/SYMFONISK.xml]: It is sold by IKEA. So IKEA could make sense at this place.
But you get Sonos everywhere in the feature XML, not IKEA.

Comment (kaikreuzer) [reply bundles/org.openhab.binding.sonos/src/main/resources/ESH-INF/thing/SYMFONISK.xml]: Well, in the description you also call it "IKEA SYMFONISK", which is imho correct. So IKEA is in the product name and they are selling it - it is only Sonos who delivery the technology for it. So I'd suggest to change the vendor here.

**Comment by lolodomo** [comment]
Wait a second, I forgot something.

**Comment by lolodomo** [comment]
Ready again.

**Pull request merged**